drivers/uktty/com/com.c

```diff
@@ -194,12 +194,8 @@ __ssz com_out(struct uk_console *dev, const char *buf, __sz len)
 
 	com_dev = __containerof(dev, struct com_device, dev);
 
-	while (leftover--) {
```

Comment:
This and the following changes regarding newline conversion. Somehow I am thinking they make more sense in the console PR.

Reply:
Let's just keep them here in this PR